vk/backend/backend.go

```diff
@@ -271,6 +286,18 @@ func (b *Backend) RunWithOutputDir(ctx context.Context, dir string) error {
 	r, err := b.run(ctx)
 	if err != nil {
 		b.readyLock.Unlock()
+		b.handleUpdate(ctx, runner.Update{
+			TaskID: b.pod.Name,
+			State:  titusdriver.Lost,
+			Mesg:   fmt.Sprintf("error launching task: %s", err.Error()),
+		})
+		err = b.writePod(ctx, dir)
+		if err != nil {
+			logger.G(ctx).WithError(err).Fatal("Unable to update pod directory")
```

Comment:
It would still "work" - TJC would show the task as failed, but with no message.  I wanted there to be a good message here, since we now fail up front if the pod is invalid.